sagemaker-triton/benchmark/pytorch/pytorch_triton_inference_recommender.ipynb

```diff
@@ -0,0 +1,786 @@
+{
```

Comment:
<p>How much does it cost to run this? Should add a note about it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3314/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>"cost of instances thats provisioned" --&gt; cost of instances that are provisioned</p><p>I think I was more concerned about that... if the default here is to launch a 9 instances and run inference tests... what's that cost?</p>